catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergConfig.java

```diff
@@ -70,6 +71,20 @@ public class IcebergConfig extends Config {
           .booleanConf()
           .createWithDefault(true);
 
+  public static final ConfigEntry<String> ICEBERG_METRICS_STORAGE =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORAGE)
+          .doc("The storage to store Iceberg metrics")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault("");
+
+  public static final ConfigEntry<Integer> ICEBERG_METRICS_STORAGE_RETAIN_DAYS =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORAGE_RETAIN_DAYS)
+          .doc("The retain days of Iceberg metrics")
+          .version("0.4.0")
+          .intConf()
+          .createWithDefault(-1);
```

Comment:
Please add this configuration entry to the configuration file and explain what the `-1` means.

Reply:
I plan to add configuration file in a separate PR, since this's not stable for now.

Reply:
add more description to configuration entry

Reply:
OK

Reply:
negative value means doesn't delete metrics